Code/GraphMol/Depictor/Wrap/rdDepictor.cpp

```diff
@@ -142,11 +142,50 @@ void GenerateDepictionMatching3DStructure(RDKit::ROMol &mol,
   RDDepict::generateDepictionMatching3DStructure(
       mol, reference, confId, referencePattern, acceptFailure, forceRDKit);
 }
+
+bool isCoordGenSupportAvailable() {
```

Comment:
If we have getPreferCoordGen, is this needed for the public API? I wonder if this should just live in the test file?

Reply:
This just tells if CoordGen is available in the RDKit build, not if it is enabled, which is what `getPreferCoordGen()` tells.  

If RDKit is not built with CoordGen support, `getPreferCoordGen()` always returns `false`. I guess we could try to guess whether CoordGen is available by playing with `setPreferCoordGen(true)`  and using `getPreferCoordGen()` to see if we actually changed the preference, but I think this is more direct.